tf_encrypted/protocol/pond.py

```diff
@@ -3215,6 +3221,76 @@ def _strided_slice_masked(prot, x: PondMaskedTensor, args: Any, kwargs: Any):
         )
 
 
+#
+# gather helpers
+#
+
+
+def _gather_public(
+    prot: Pond, x: PondPublicTensor, indices: list, axis: int = 0
+) -> List[PondPublicTensor]:
+
+    x_on_0, x_on_1 = x.unwrapped
+
+    with tf.name_scope("gather"):
+
+        with tf.device(prot.server_0.device_name):
+            y0_g = x_on_0.gather(indices, axis=axis)
```

Comment:
consider renaming to `y_on_0_g` for consistency (public value vs share)

Reply:
good point!